main.nf

```diff
@@ -458,11 +459,13 @@ process VEP_ANNOTATE {
     """
 }
 
-process FEATURE_ENGINEERING_PART1 {
+process DB_ANNOTATE {
```

Comment:
ANNOTATE_VARIANTS

Reply:
DB_ANNOTATE is vague

Reply:
I agree but this process indeed annotates hundreds of features by referring 5+ databases.
How should we describe this process's behavior? I will ask gpt about this haha.

Reply:
I would like to suggest MODULE_ANNOTATE or ANNOTATE_BY_MODULES